packages/shared/src/lib/featureManagement.ts

```diff
@@ -33,6 +34,10 @@ const feature = {
     webm: cloudinary.onboarding.video.webm,
     mp4: cloudinary.onboarding.video.mp4,
   }),
+  postPageOnboarding: new Feature(
+    'post_page_onboarding',
```

Comment:
@nensidosari for me to remember to setup the experiment

Reply:
experiment feature is setup as well

Reply:
Thank you!